migrations/8_setup_meta_colony.js

```diff
@@ -9,6 +9,8 @@ const ITokenLocking = artifacts.require("./ITokenLocking");
 const EtherRouter = artifacts.require("./EtherRouter");
 const Token = artifacts.require("./Token");
 
+const DEFAULT_STAKE = "24000000000000000000000";
```

Comment:
Why is this default stake different to the default for tests?

Reply:
No real reason, the build was running when I noticed this discrepancy and didn't want to restart. I can update.